docs/content/components/label.md

```diff
@@ -31,7 +31,7 @@ Use `aria-label` if the `Label` or the context around it don't explain the label
 
 ### Schemes
 
-<Example src="<span data-view-component='true' class='Label Label--large'>Default</span><span data-view-component='true' class='Label Label--primary Label--large'>Primary</span><span data-view-component='true' class='Label Label--secondary Label--large'>Secondary</span><span data-view-component='true' class='Label Label--accent Label--large'>Accent</span><span data-view-component='true' class='Label Label--success Label--large'>Success</span><span data-view-component='true' class='Label Label--attention Label--large'>Attention</span><span data-view-component='true' class='Label Label--danger Label--large'>Danger</span><span data-view-component='true' class='Label Label--severe Label--large'>Severe</span><span data-view-component='true' class='Label Label--done Label--large'>Done</span><span data-view-component='true' class='Label Label--sponsors Label--large'>Sponsors</span>" />
+<Example src="<span data-view-component='true' class='Label'>Default</span><span data-view-component='true' class='Label Label--primary'>Primary</span><span data-view-component='true' class='Label Label--secondary'>Secondary</span><span data-view-component='true' class='Label Label--accent'>Accent</span><span data-view-component='true' class='Label Label--success'>Success</span><span data-view-component='true' class='Label Label--attention'>Attention</span><span data-view-component='true' class='Label Label--danger'>Danger</span><span data-view-component='true' class='Label Label--severe'>Severe</span><span data-view-component='true' class='Label Label--done'>Done</span><span data-view-component='true' class='Label Label--sponsors'>Sponsors</span>" />
```

Comment:
i don't know why this file has changed, honestly. but it keeps showing up as modified on my local system whenever i do any work so i figured it should be committed. let me know if these changes should be dropped, tho

Reply:
I'm not sure. Maybe we just commit this diff and see what happens? 😂

Reply:
@joelhawksley honestly, i'm a big fan of that approach! once at a job almost 20 years ago, the IT Director found a box connected to a giant cable coming out of a hole in a wall, under a folding table in a closet. literally just a hole roughly cut out of the wall... no one knew what it did, so he unplugged it. three days later, the order entry people were wondering why they had not received any new orders in three days. turns out that box connected customer orders to the order entry team. oops 🤣